packages/engine/bin/cli/src/experiment.rs

```diff
@@ -95,75 +105,68 @@ async fn run_experiment_with_manifest(
     };
     engine_process
         .send(&proto::EngineMsg::Init(init_message))
-        .await?;
-    log::debug!("Sent init message to {}", &experiment_id);
+        .await
+        .context("Could not send `Init` message")?;
+    debug!("Sent init message to {experiment_id}");
 
     loop {
         let msg: Option<proto::EngineStatus>;
         tokio::select! {
             _ = time::sleep(*ENGINE_WAIT_TIMEOUT) => {
-                log::error!("Did not receive status from experiment {} for over {} seconds. Exiting now.", &experiment_id, ENGINE_WAIT_TIMEOUT.as_secs());
+                error!("Did not receive status from experiment {experiment_id} for over {:?}. Exiting now.", *ENGINE_WAIT_TIMEOUT);
```

Comment:
How come we're removing the `log::` prefix? I don't feel strongly about it, just wondering the reason

Reply:
We have 
```rust

#[macro_use]
extern crate log;
```

In the top level module, which imports the macros into scope for the whole. I like this for logging, but since Rust 2018 it's unidiomatic.

https://stackoverflow.com/questions/50999749